cypress/integration/single-beacon-owner/i-can-register-a-beacon-with-many-uses.spec.ts

```diff
@@ -58,12 +65,15 @@ import {
 } from "../common/selectors-and-assertions.spec";
 
 describe("As a single beacon owner with many uses,", () => {
-  it("I can register my beacon for a maritime and an aviation use", () => {
+  it.only("I can register my beacon for a land, maritime pleasure, and aviation pleasure use", () => {
```

Comment:
Updated to test registering a beacon with land, maritime pleasure, and aviation pleasure use